bool create_scene
bool clean_scene
point_cloud_segmentation/GraspObject obj_seg
geometry_msgs/PoseStamped object_pose_world
string object_mesh_path
---
bool success
